postgraphiql/src/components/PostGraphiQL.js

```diff
@@ -94,14 +95,19 @@ class PostGraphiQL extends React.PureComponent {
     // stuffs.
     schema: null,
     showHeaderEditor: false,
+
+    // deinspanjer: I think we'd like to have a way to wait for an initial value
+    // if we are using graphiqlAuthorizationEventOrigin but I'm not sure how to do that.
     headersText: '{\n"Authorization": null\n}\n',
```

Comment:
I think reading it from the query hash would be a good solution (and would solve other people's problems without requiring `graphiqlAuthorizationEventOrigin` also). e.g. `http://localhost:5000/graphiql#token=skldjfojweoij0goerng0iergn`

```suggestion
    headersText: headersFromLocationHash ? JSON.stringify(headersFromLocationHash) : '{\n"Authorization": null\n}\n',
```

Where

```js
const qs = require('qs');
const hashObject = qs.parse(window.location.hash || "");
const token = hashObject.token;
const headersFromLocationHash = token ? {Authorization: `Bearer ${token}`} : null;
```

Reply:
Great, in my private build I have this hash parsing.  I want to change it to be `Bearer=` and I can submit a separate PR for that.